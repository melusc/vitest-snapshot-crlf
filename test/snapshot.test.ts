import {test, expect} from 'vitest';

test.for([
	'a\nb',
	'a\r\nb',
])('"%s".split("")', (input) => {
	expect(input.split("")).toMatchSnapshot();
})
