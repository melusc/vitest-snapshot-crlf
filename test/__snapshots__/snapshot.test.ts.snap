// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`"a
b".split("") 1`] = `
[
  "a",
  "
",
  "
",
  "b",
]
`;

exports[`"a
b".split("") 1`] = `
[
  "a",
  "
",
  "
",
  "b",
]
`;
